html/Ellipsis/index.html

```diff
@@ -0,0 +1,233 @@
+          product management to create exceptional products. 
+          We start each project by understanding your business goals, 
+          the user’s needs and the product requirements. 
+          Then we help translate them into effective and elegant solutions.
+        </p>
+      </div>
+    </section>
+
+    <!-- Services -->
+    <section class="services">
+      <!-- Header -->
+      <header class="services__title">
+        services
+      </header>
+
+      <!-- Boder top line -->
+      <div class="services__line"></div>
+      
+      <!-- Description -->
+      <div class="services__description">
+          <p>
+            Before we sign a contract or write a line of code our team will spend the necessary time needed to understand your product vision. 
+            <br>
+            The outcome of this scoping session is a proposal or high-level statement of work. 
+            Based on your requirements, we can provide a complete offering or just the pieces you need. 
+          </p>
+      </div>
+    </section>
+
+    <!-- What we do -->
+    <section class="what-we-do">
+      <!-- Header -->
+      <header class="what-we-do__title">
+        what we do
+      </header>
+
+      <!-- Design -->
+      <div class="what-we-do__action">
+        <!-- Action image -->
+        <picture class="services__icon">
+          <source media="(min-width: 1200px)" srcset="./assets/design/shape-2@3x.png">
+          <source media="(min-width: 768px)" srcset="./assets/design/shape-2@2x.png">
+          <img src="./assets/design/shape-2.png" alt="Design">
+        </picture> 
+
+        <!-- Title -->
+        <h4 class="action__title">
+          design
+        </h4>
+
+        <!-- Description -->
+        <p class="action__description">
+          Our design practice offers a full range of services including branding strategy,
+          interaction and visual design and user experience testing. 
+          <br>
+          Throughout your project, our designers create and implement visual designs and workflows, solicit user feedback, and work with you to make sure what gets built is what is needed. 
+        </p>
+      </div>
+
+      <!-- Development -->
+      <div class="what-we-do__action">
+        <!-- Action image -->
+        <picture class="services__icon">
+          <source media="(min-width: 1200px)" srcset="./assets/development/shape-3@3x.png">
+          <source media="(min-width: 768px)" srcset="./assets/development/shape-3@2x.png">
+          <img src="./assets/development/shape-3.png" alt="Development">
+        </picture> 
+  
+        <!-- Title -->
+        <h4 class="action__title">
+          development
+        </h4>
+  
+        <!-- Description -->
+        <p class="action__description">
+          Our design practice offers a full range of services including branding strategy,
+          interaction and visual design and user experience testing. 
+          <br>
```

Comment:
please avoid using br except poems and addresses
https://stackoverflow.com/a/1726103
use `<p>` for splitting by paragraphs

Reply:
and please take care of every entry of `br`